app/views/trains/_form.html.slim

```diff
@@ -3,26 +3,26 @@
   p
     = f.label :name
     br
-    = f.text_field :name, value: train["name"]
+    = f.text_field :name, value: train ? train["name"] : ""
```

Comment:
you shouldnt need this, i think if you instantiate a new object to behave like the train response you wont need all these ternaries, they arent very nice to have littered around the app